src/io/xml.c

```diff
@@ -41,7 +41,7 @@
     "tests=\"" CR_SIZE_T_FORMAT "\" "       \
     "failures=\"" CR_SIZE_T_FORMAT "\" "    \
     "errors=\"" CR_SIZE_T_FORMAT "\" "      \
-    "disabled=\"" CR_SIZE_T_FORMAT "\""
+    "skipped=\"" CR_SIZE_T_FORMAT "\""
```

Comment:
"skipped" is not a valid field for `testsuites` (although it exists for `testsuite`) in the [JUnit 4 XML spec](https://raw.githubusercontent.com/behave/behave/master/etc/junit.xml/junit-4.xsd)